File: javascript/hulluxhd/football-points.js
function footballPoints(wins, draws, losses) {
    const points = {
        wins: 3,
        draws: 1,
        losses: 0
    }

    const pointsSum = points.wins * wins + points.draws * draws + points.losses * losses

    return `O time fez, ao todo, ${pointsSum} pontos.`
}

console.log(footballPoints(3, 4, 2))
console.log(footballPoints(5, 0, 2))
console.log(footballPoints(0, 0, 1))

File: java/cpazzz/FootballPoints.java
public class FootballPoints
{

    public static void main(String[] args)
    {
        int wins = 3;
        int draws = 4;
        int losses = 2;

        System.out.println(calcPoints(wins, draws, losses));
    }

    public static int calcPoints(int w, int d, int l)
    {
        int points = (w * 3) + d;
        return points;
    }

}
